web_boolean_button/static/tests/form_tests.js

```diff
@@ -0,0 +1,312 @@
+odoo.define("web_boolean_button.form_tests", function(require) {
+    "use strict";
+
+    /* global QUnit*/
+
+    var FormView = require("web.FormView");
+    var testUtils = require("web.test_utils");
+    var createView = testUtils.createView;
+
+    QUnit.module("fields", {}, function() {
```

Comment:
@SkiBY nice work so far! I wish I could approve directly but I still have something to complain about this test:

- When I go to the runbot (Go to first page of PR with our conversation, go to the button, go to one of four checks which is called "Runbot", click the link, click on "connect", end up [here](http://3469276-1822-e928bb.runbot1-3.odoo-community.org/), login as admin, go to `/web/tests`, I cannot find the test for `web_boolean_button`. I think that's because you called this test `fields`. This comes from Odoo 12.0 since there, it's a test that tests all fields, but here it should only test the widget. I think if you just start directly with `QUnit.module("web_boolean_button", ...` such as in [this example](https://github.com/OCA/web/blob/13.0/web_widget_dropdown_dynamic/static/tests/web_widget_dropdown_dynamic_tests.js#L11), and you just leave the last part of your tests, then in your local it will appear in `/web/tests` and it will run. If it runs correctly there, then you can push and I will also see it run correctly here, and I can approve :-)

Reply:
Thank you, @thomaspaulb
Naming is changed